EXAMPLE.md

```diff
@@ -0,0 +1,302 @@
+later, possibly dropping `-cdrom opensuse.iso` part):
+```
+qemu-system-x86_64 -m 6G \
+                   -cdrom opensuse.iso \
+                   -machine q35,accel=kvm \
+                   -drive if=virtio,file=disk.qcow2,format=qcow2 \
+                   -cpu host \
+                   -smp `nproc` \
+                   -netdev user,id=mynet0,hostfwd=tcp:127.0.0.1:4151-:22,hostfwd=tcp:127.0.0.1:4150-:80 \
+                   -device virtio-net,netdev=mynet0
+```
+
+### Installation Process
+
+The installer is quite standard and easy to use, so won't go into much detail.
+
+1. Pick "Installation" in boot menu, booting and then starting the installer
+   will take some time as it downloads packages.
+2. Accept license, probably keep English locale to keep it simple.
+3. Agree to activate "Online Repositories", then press "Next" to actually do it
+   (default state of the list seems sensible).  Wait for a bit.
+4. Pick "Server" for system role as we don't need any desktop environment.
+5. Accept default partitioning (BTRFS with a bunch of subvolumes) or adjust to
+   something like: boot partition, EXT4 root and no swap.
+6. Pick a time zone.  Can use localtime to avoid time difference in Web-UI.
+7. Don't bother creating a regular user.
+8. Pick root's password.
+9. Installation summary:
+   * Can disable CPU Mitigations in "Security" section for slightly better
+     performance.
+   * SSH should already be enabled in "Security" section.
+10. Hit "Install" and come back in about 5 minutes.
+
+### First Boot
+
+Connect via SSH:
+```
+ssh -p 4151 root@127.0.0.1
+```
+
+Disable AppArmor right away (alternative is to configure it properly for the
+worker to give it access to things like `/usr/bin/tar`; mind that skipping
+AppArmor during installation won't help because it will be installed later as a
+dependency):
+```
+systemctl mask --now apparmor
+```
+
+### openQA Setup
+
+Install both server and worker (about 600 packages in total):
+```
+zypper install -y openQA openQA-worker
+```
+
+Setup Apache reverse proxy:
+```
+/usr/share/openqa/script/configure-web-proxy
+systemctl enable apache2
+systemctl restart apache2
+```
+
+Either disable `firewalld` through `systemctl disable firewalld` or allow access
+to TCP port 80:
+```
+firewall-cmd --zone=public --add-port=80/tcp
+firewall-cmd --zone=public --add-port=80/tcp --permanent
+```
+
+Configure authorization in `/etc/openqa/openqa.ini` (`Fake` should be fine as
+long as the server is not publicly accessible):
+```
+[auth]
+method = Fake
+```
+
+Start everything:
+```
+systemctl enable --now openqa-webui
+systemctl enable --now openqa-scheduler
+systemctl enable --now openqa-worker@1
+```
+
+At this point openQA should be accessible at <http://127.0.0.1:4150> and "Login"
+link should activate "Demo" account.
+
+Open the main menu (top-right corner) and select "Manage API keys"
+(<http://127.0.0.1:4150/api_keys>).  Remove `1234567890ABCDEF` key as it will
```

Comment:
AFAIR this key is extended each time Demo user logs in.

Reply:
Might be, but I don't think it's extended for a long time (a day?) as it expired for me, simpler to make a new one right away.